weatherman/constants.js

```diff
@@ -0,0 +1,27 @@
+
+export const WEATHER_FILE_HEADERS = {
```

Comment:
we dont need the headers that are not being used

Reply:
these headers are used for index mapping

Reply:
we can only store the ones that we need to calculate, this is extra

Reply:
resolved